### PR TITLE
Fixes 4539 : When Wikidata has no description showing instance_of's label as a description

### LINKED ISSUE #4539: When Wikidata has no decription, show class label instead
1. Using the app to upload a picture, search for 潮騒
2. Only the caption shows up, as item https://www.wikidata.org/wiki/Q60985286 has no description in any language.

In such cases, it would be great to show the label of the Wikidata item that Q60985286 is an instance of (https://www.wikidata.org/wiki/Q105543609 "musical work/composition").

It would be a great help in this case as there are many homonyms (novels, movies, bridges, sculptures, etc).

### PR DESCRIPTION
**Pull request by Ayan-10**
**Description (required)**

Fixes #4539

What changes did you make and why?

Some Wikidata has no description. So, I used instance of Wikidata's label as a description.
While searching for Wikidata's created another call for getting labels of instance of Wikidatas and modified the Wikidata's description.

**Tests performed (required)**

Tested latest master ProdDebug on Pixel 3 with API level 30.

**Screenshots (for UI changes only)**

### CODE REVIEW

**Comment by Ayan-10** [comment]
@nicolas-raoul Hope it works. If it's working how about showing thumbnails in the same way?

**Comment by Ayan-10** [comment]
Thanks for the detailed explanation. I get the point.

File: app/src/main/java/fr/free/nrw/commons/explore/depictions/DepictsClient.kt [hunk 1]
```diff
@@ -38,7 +43,45 @@ class DepictsClient @Inject constructor(private val depictsInterface: DepictsInt
...
+                if (entity.descriptions().byLanguageOrFirstOrEmpty() == "") {
+                    val entities: Entities = getEntities(entity[WikidataProperties.INSTANCE_OF]
+                        .toIds()[0]).blockingGet()
+                    val nameAsDescription = entities.entities().values.first().labels()
+                        .byLanguageOrFirstOrEmpty()
+                    DepictedItem(entity,
+                        entity.labels().byLanguageOrFirstOrEmpty(),
+                        nameAsDescription)
+                } else {
+                DepictedItem(entity,
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/explore/depictions/DepictsClient.kt:63]: Indentation is a bit off it seems.

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]
Working great!
Would you mind adding unit tests to cover the uncovered parts?
Thanks!

**Review by nicolas-raoul — APPROVED** [review]

**Pull request merged**